ast/compile.go

```diff
@@ -319,7 +320,30 @@ func (c *Compiler) WithBuiltins(builtins map[string]*Builtin) *Compiler {
 	return c
 }
 
-// WithUnsafeBuiltins will add all built-ins in the map to the "blacklist".
+// WithCapabilities sets capabilities to enable during compilation. Capabilities allow the caller
+// to specify the set of built-in functions available to the policy.
```

Comment:
Is it worth mentioning that this does not affect any evaluation of the compiled policies? I guess for people who know what this stuff is maybe its pretty obvious.. but if someone made a compiler and specified some capabilities, then evaluated with the compiler, the results may be surprising as its using the current builtin implementations with the (potentially) different schemas defined by the capabilities that compiled the policies.

Reply:
I'll include a note about this only being used at compile-time and that it's assumed the policies are evaluated by a compatible version of OPA.